src/components/navbar.tsx

```diff
@@ -64,7 +64,7 @@ function NavBar() {
               <NavbarBrand/>
             </Navbar.Brand>
             <Nav className="me-auto justify-content-end flex-grow-1 pe-3">
-              {data.map((item: MenuItem) => (item.drop_down_menu_list.length > 0 ? gen_custom_dropdown(item, show, setShow): gen_custom_button(item)))} 
+              {data.map((item: MenuItem, index: Number) => (item.drop_down_menu_list.length > 0 ? gen_custom_dropdown(item, index, show, setShow): gen_custom_button(item, index)))} 
```

Comment:
where is this index number coming from?

Reply:
The callback returns it. Map is a very common javascript function, I suggest you do a little research and look at the docs - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

![image](https://user-images.githubusercontent.com/13427383/203483794-15d6f335-fa8b-4588-9e2d-088492ec3cc8.png)